sdk/eventgrid/azure-eventgrid/azure/eventgrid/_helpers.py

```diff
@@ -141,9 +141,14 @@ def _cloud_event_to_generated(cloud_event, **kwargs):
         **kwargs
     )
 
-def _from_cncf_events(event):
+def _from_cncf_events(event): # pylint: disable=inconsistent-return-statements
```

Comment:
just curious but, is this generated code? if not, is it not annotated b/c it's internal?

Reply:
not generated code!